charts/datadog/README.md

```diff
@@ -29,7 +29,6 @@ Kubernetes 1.10+ or OpenShift 3.10+, note that:
 | Repository | Name | Version |
 |------------|------|---------|
 | https://helm.datadoghq.com | datadog-crds | 0.4.7 |
-| https://prometheus-community.github.io/helm-charts | kube-state-metrics | 2.13.2 |
```

Comment:
is it attended that we remove this line even if we just disable the default installation?